proto/select.proto

```diff
@@ -65,7 +65,17 @@ message SelectRequest {
 	repeated Expr aggregates = 13;
 
 	// time zone offset in seconds
-	optional int64 time_zone_offset = 14;
+	optional int64 time_zone_offset = 14 [(gogoproto.nullable) = false];
+
+	/* flags is used to store flags that change the execution mode, it contains:
```

Comment:
All the comment in this file is prefixed will `//`, I think here should follow the convention too.